statistics/queries/continent_country_region_histogram.sql

```diff
@@ -0,0 +1,361 @@
+WITH
+# Select the initial set of download test results
+dl_per_location AS (
+  SELECT
+    test_date,
+    client.Geo.continent_code AS continent_code,
+    client.Geo.country_code AS country_code,
+    client.Geo.country_name AS country_name,
+    CONCAT(client.Geo.country_code, '-', client.Geo.region) AS ISO3166_2region1,
+    a.MeanThroughputMbps AS mbps,
+    NET.SAFE_IP_FROM_STRING(Client.IP) AS ip
+  FROM `measurement-lab.ndt.unified_downloads`
+  WHERE test_date = @startday
```

Comment:
Could we change this to read `WHERE test_date BETWEEN @startdate AND @enddate`? Same for the `ul_per_location` subquery.

As discussed on separate channels, this should be logically equivalent in terms of how the histogram is generated (on a daily basis) but - for reasons unknown to me - BigQuery seems to be ~6x faster when operating on a range of dates.